test/visibility/private/moduleSymbols/transitiveUsePrivate.future

```diff
@@ -0,0 +1,6 @@
+Bug
```

Comment:
IMO this is supposed to be just two lines (but that has changed over time, which is why other futures have more) -- see also https://chapel-lang.org/docs/developer/bestPractices/TestSystem.html#a-test-that-tracks-a-failure

I'd expect 

```
design: transitive 'use' can allow access to private symbols
#20707
```

Of course you could word it as 

```
bug: A transitive 'use' can incorrectly allow access to private symbols in its scope
#20707
```

(but I'd call it a design question until we reach agreement on the issue).

Reply:
Ah, I interpreted that document incorrectly. Will update it, and label as `design` as well